client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -117,10 +117,18 @@ export const BatchTable: FC<StocktakeLineEditTableProps> = ({
     [
       expiryDateColumn,
       {
-        width: 100,
+        width: 120,
         setter: patch => update({ ...patch, countThisLine: true }),
       },
     ],
+    {
+      key: 'comment',
+      label: 'label.stocktake-comment',
```

Comment:
would you mind tidying the column headers too please?

<img width="587" alt="image" src="https://user-images.githubusercontent.com/9192912/212760774-4afccc60-7ad8-42e4-8e1c-3b6e0907d224.png">


I wondered if we could get away with `Snapshot Packs` and `Counted Packs` - and adding a description field to the columns too, so that you can get a tooltip of the full title